src/helpers/storage.ts

```diff
@@ -0,0 +1,18 @@
+import type { IStoredKeys } from '../routes/connect/IStoredKeys';
+import StorageKeyNotFoundError from '../routes/connect/errors/StorageKeyNotFoundError';
+
+export function storageData(key: string, cryptoKey: string): void {
+    sessionStorage.setItem('cryptoKey', cryptoKey);
+    sessionStorage.setItem('privateKey', key);
+}
+
+export function getStorageData(): IStoredKeys {
+    const secretKey = sessionStorage.getItem('privateKey');
+    const cryptoKey = sessionStorage.getItem('cryptoKey');
+
+    if (secretKey && cryptoKey) {
+        return { privateKey: secretKey, cryptoKey: cryptoKey };
+    } else {
+        throw new StorageKeyNotFoundError();
+    }
+}
```

Comment:
I just realised all of this doesn't belong in the storage helper. The storage helper shouldn't know what is it storing.